maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -389,7 +470,8 @@ private void setupDatabaseForServer(Server server) throws MojoExecutionException
 
 		if (server.isMySqlDb() || server.isPostgreSqlDb()) {
 			String uri = getUriWithoutDb(server);
-			try (DBConnector connector = new DBConnector(uri, server.getDbUser(), server.getDbPassword(), server.getDbName())) {
+			try (DBConnector connector = new DBConnector(uri, server.getDbUser(), server.getDbPassword(),
+					server.getDbName())) {
```

Comment:
Are these just formatting changes?

Reply:
Yeah, I used Intlij to reformat the file.